supabase/migrations/20240424132005_add_manager_and_staff_roles_and_rename_caller_to_volunteer.sql

```diff
@@ -0,0 +1,65 @@
+drop view if exists "public"."profiles_plus";
+
+alter type "public"."role" rename to "role__old_version_to_be_dropped";
+
+create type "public"."role" as enum ('volunteer', 'admin', 'manager', 'staff');
+
+alter table "public"."profiles" alter column role type "public"."role" using (
+      CASE role::text
+        WHEN 'caller' THEN 'volunteer'
+        ELSE role::text
+      END
+    )::"public"."role";
+
+drop type "public"."role__old_version_to_be_dropped";
+
+create or replace view "public"."profiles_plus" as  SELECT users.id AS user_id,
+    users.email,
+    users.created_at,
+    users.updated_at,
+    profiles.first_name,
+    profiles.last_name,
+    profiles.role,
+    profiles.telephone_number
+   FROM (auth.users
+     LEFT JOIN profiles ON ((users.id = profiles.user_id)))
+  ORDER BY profiles.first_name;
```

Comment:
Is this exactly the same as the existing definition? Did we need this to make the role work just out of curiosity?

Reply:
This was automatically generated when making the migration file via the diff method. I think we need to drop and then recreate the view (yes its exactly the same) just because the role enum changed.

Reply:
fair enough!